apps/web/src/services/asset-inventory/components/SecurityLSB.vue

```diff
@@ -89,11 +91,12 @@ const routeToFirstCloudServiceType = async () => {
 };
 
 /* Watchers */
-watch(() => state.pageParams, (pageParams) => {
+watch(() => state.pageParams, async (pageParams) => {
     if (pageParams?.name) {
-        securityPageStore.setSelectedCloudServiceType(pageParams.group, pageParams.name);
+        await securityPageStore.setSelectedCloudServiceType(pageParams.group, pageParams.name);
+        await cloudServiceDetailPageStore.setProviderGroupName(pageParams);
     } else {
-        securityPageStore.setSelectedCloudServiceType();
+        await securityPageStore.setSelectedCloudServiceType();
```

Comment:
question: no need to reset provider group name?🧐

Reply:
The 'AssetInventoryContainer.vue' file is being reset when it is unmounted!